Makefile

```diff
@@ -0,0 +1,25 @@
+VERSION?=$(shell git describe --tags --dirty=-dirty)
+BP_UNDER_TEST?="../java-buildpack.cnb"
+
+clean:
+	rm -fr target
+	rm -f *.cnb
+	mvn clean -f integration/pom.xml
+
+create-package:
```

Comment:
Should we rather use bash scripts in a `scripts` folder?

Reply:
Personally, I think it's easier to inline stuff. I will typically only put a script if I think there is a need to run the script manually or if it's being invoked in a bunch of different places (DRY).

Reply:
> Personally, I think it's easier to inline stuff. I will typically only put a script if I think there is a need to run the script manually or if it's being invoked in a bunch of different places (DRY).

Sounds like a good rule of thumb. In this case, I was rather worried about the usage of a `Makefile` - it is rather rare for buildpacks, isn't it? Most other buildpacks I had contact so far used some scripts in the `scripts` folder.

Imho, I would prefer a `Makefile`, but using scripts consistently in all buildpacks has it charm.

Reply:
I'm not sure that we ever logged an issue, but we've discussed moving the Java-related buildpacks to use Makefiles. Using Makefiles is the common approach in Go development, so we want to follow that. Basically, what is setup here -> https://github.com/buildpacks/profile.

We probably need a RFC for that though, and it should probably be something we try to do project-wide, not just in the Java buildpacks.